src/fgimic/jfield.f90

```diff
@@ -367,6 +367,7 @@ subroutine jvector_plots(this,tag)
         else if ( ((trim(this%grid%mode)=='base' .or. trim(this%grid%mode)=='std') .and. this%grid%gauss) &
               .or. trim(this%grid%mode)=='file' ) then ! (base && gauss) || file
           allocate(jval_unstructured(this%grid%npts(1), 3))
+          allocate(this%sigmaval(this%grid%npts(1), 3))
```

Comment:
Why did you add this to the type structure for the J tensor? Outside the property routine this entry will not be needed.

Reply:
I am sorry if this is wrong I have limited knowledge of Fortran. 
The variable sigmaval was defined at line 27 in the data type jfield_t, which was called in the subroutine jvector_plots as 'this' at line 256

Reply:
No need to be sorry. I suggest to avoid adding extra fiels to fix type structures. We need this only in the property routine.